### ISSUE TITLE
New package: Jitsi  Desktop 2.10.5550

### ISSUE THREAD

**Issue by gorkawien** [void-linux/void-packages#23024]
Jitsi is an audio/video Internet phone and instant messenger written in Java. It supports some of the most popular instant messaging and telephony protocols such as SIP, Jabber/XMPP (and hence Facebook and Google Talk), AIM, ICQ, MSN, Yahoo! Messenger.

The development of Jitsi started at the University of Strasbourg, France. Originally the project was known as SIP Communicator. Throughout the years our community has grown to include members and contributors from Brazil, Bulgaria, Cameroon, China, Estonia, France, Germany, India, Japan, Romania, Spain, Switzerland, UK, USA, and others.

Jitsi is based on the OSGi architecture using the Felix implementation from Apache. This makes it very extensible and particularly developer friendly.

https://github.com/jitsi/jitsi/releases/tag/Jitsi-2.10

**Comment by dm17** [comment]
Does Gajim have everything you need in Jitsi? It is already on Void and seems to be a bit higher quality (on Linux at least).

**Comment by gorkawien** [comment]
@dm17 It could be also a good idea to try Gajim. I have one question: Which package should I use for videoconferencing?
farstream or telepathy-farstream??

**Comment by dm17** [comment]
@gorkawien I haven't tried it yet, but I assumed the gajim pkg.

**Comment by gorkawien** [comment]
@dm17 yes I saw that farstream is included in Gajim. Sometimes I have to be more patient.
